.github/workflows/build.yml

```
@@ -0,0 +1,189 @@
+name: build
+
+on:
+  push:
+    branches: [ master, '[1-9][0-9].[0-9]' ]
+  pull_request:
+    branches: [ master, '[1-9][0-9].[0-9]' ]
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip travis]')"
+    name: "PHP ${{ matrix.php-versions }} Test ${{ matrix.static-analysis != 'no' && matrix.static-analysis || '' }} (deps: ${{ matrix.dependencies }})"
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.3', '7.4']
+        dependencies: ['highest']
+        composer-arguments: [''] # to run --ignore-platform-reqs in experimental builds
+        static-analysis: ['no']
+        experimental: [false]
+        include:
+          - php-versions: '7.3'
+            static-analysis: 'psalm'
+            experimental: false
+            dependencies: 'highest'
+
+          # Experimental build for PHP 8
+          #- php-versions: '8.0'
+          #  composer-arguments: '--ignore-platform-reqs'
+          #  static-analysis: 'no'
+          #  experimental: true
+          #  dependencies: 'highest'
+
+          # Build for minimum dependencies. Fails right now, hence deactivated.
+          #- php-versions: '7.3'
+          #  static-analysis: 'no'
+          #  experimental: false
+          #  dependencies: 'lowest'
+
+    runs-on: ubuntu-latest
+    services:
+      mariadb:
+        image: mariadb:10.2
+        env:
+          MYSQL_USER: neos
+          MYSQL_PASSWORD: neos
+          MYSQL_DATABASE: flow_functional_testing
+          MYSQL_ROOT_PASSWORD: neos
+        ports:
+          - "3306:3306"
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+      postgres:
+        image: postgres:9.5
+        env:
+          POSTGRES_USER: neos
+          POSTGRES_PASSWORD: neos
+          POSTGRES_DB: flow_functional_testing
+        ports:
+          - "5432:5432"
+        options:  --health-cmd=pg_isready --health-interval=10s --health-timeout=5s --health-retries=3
+
```

```suggestion
      redis:
        image: redis:latest
        ports:
          - "6379:6379"
        options: --health-cmd "redis-cli ping" --health-interval 10s --health-timeout 5s --health-retries 5
      memcached:
        image: memcached:latest
        ports:
          - "11211:11211/udp"
        options: --health-cmd "timeout 5 bash -c 'cat < /dev/null > /dev/udp/127.0.0.1/11211'" --health-interval 10s --health-timeout 5s --health-retries 5

```
Add redis and memcached services.